docs/general/installation/container.md

```diff
@@ -311,28 +311,35 @@ SuccessExitStatus=0 143
-
-   1. Go to Apps page from the top level SCALE menu
-   2. Select Manage Catalogs tab on the Apps page
-   3. Click Add Catalog
-   4. After reading the iXsystems notice, click Continue and enter the required information:
-      Name: truecharts
-      Repository: `https://github.com/truecharts/catalog`
-      Preferred Trains: `enterprise` and `stable`
-      Branch: main
-   5. Click Save and allow SCALE to refresh its catalog with TrueCharts (this may take a few minutes)
-
-2. Click `Available Applications` and search for `Jellyfin`
-
-3. Click `Install`, which will take you to the GUI Wizard and you'll be able to fill out the necessary info
-
-   - Server URL to publish in UDP Auto Discovery response.
-   - Networking, Ingress (Reverse Proxy), Security Options
-   - Adding Storage (for media folders) is also a standalone guide available in the [TrueCharts documentation](https://truecharts.org/manual/SCALE/guides/add-storage/). For Jellyfin the recommendation is to add storage as `Additional App Storage`
-
-4. Click Save and once it's up and running you'll be able to click Open to access `Jellyfin`.
+## TrueNAS SCALE
+
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App in the official app catalog with direct integration into the GUI, no CLI needed.
+
+1. Go to Apps page from the top level SCALE menu
+
+2. Click `Discover Apps` and search for `Jellyfin`
+
+3. Click `Install`, which will take you to the GUI installation wizard and you'll be able to fill out the necessary info
+
+   - Published Server URL to publish in UDP Auto Discovery response
+   - User and Group Configuration: adjust only if needed, defaults to `568:568` which is `apps:apps` user/group
+   - Network Configuration
+     - Tick `Host Network` if DLNA is used, otherwise it's optional.
+     - You may optionally change the `WebUI Port` to the Jellyfin's default of `8096` if it's not occupied already by another app.
+   - Storage Configuraton
+     - For `Jellyfin Config Storage` and `Jellyfin Cache Storage`, consider mapping Host Paths for ease of backup.
+       - Store them on SSD storage if possible, as using HDD storage **will** lead to poor experience.
+     - For `Jellyfin Transcode Storage`, consider using `Temporary` or `tmpfs`
+       - `Temporary` places a Docker volume under your configured Apps dataset (`Apps` -> `Configuration` -> `Choose Pool`). Avoid if that's on HDD.
+       - `tmpfs` creates a temporary directory on the RAM. Ensure you set the limit to at least a few gigabytes and that you have considerate amount of spare memory, since TrueNAS disables swap by default. You may need to increase the limit if you have many large media files streamed with transcoding at once, or transcodes might fail. Note that total memory usage may also be capped in the `Resources Configuration` section.
```

Comment:
```suggestion
       - `tmpfs` creates a temporary directory on the RAM. Ensure you set the limit to at least a few gigabytes, and that you have considerate amount of spare memory, since TrueNAS disables swap by default. You may need to increase the limit if you have many large media files streamed with transcoding at once, or transcodes might fail. Note that total memory usage may also be capped in the `Resources Configuration` section.
```